firefox-ios/Client/TabManagement/Legacy/LegacyTabManager.swift

```diff
@@ -571,9 +570,6 @@ class LegacyTabManager: NSObject, FeatureFlaggable, TabManager, TabEventHandler
         self.removeTab(tab, flushToDisk: true)
         self.updateIndexAfterRemovalOf(tab, deletedIndex: index, viableTabsIndex: viableTabsIndex)
 
-        // TODO: FXIOS-9084 This is not ideal, follow up in this ticket to make tab selection reasonably synchronous
-        try? await Task.sleep(nanoseconds: NSEC_PER_SEC/10)
-
```

Comment:
@OrlaM I think this is okay to remove now? Let me know if not though, I might have misunderstood.